cfgBuilder/cfgBuilder_test.go

```diff
@@ -18,6 +18,7 @@ import (
 
 const goerliEndpoint = "http://goerli.com"
 const kottiEndpoint = "http://kotti.com"
+const golangEndpoint = "https://golang.org/"
```

Comment:
how do we choose endpoints?

Reply:
Doesn't really matter for tests, any valid url is fine.